packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
             created
           example: 1587667174725
 
+    vod-asset:
+      type: object
+      table: vod_asset
+      required:
+        - name
```

Comment:
Should also have `additionalProperties: false`. Please add to **every object** on the schema unless there is need for it.